### ISSUE TITLE
[BUG] - can't login via https to ampache server / no playback when using 'low security' checkbox

### ISSUE THREAD

**Issue by i-vis** [CappielloAntonio/tempo#295]
## Describe the bug
1) Attempting to connect to ampache server with https://ampache.tld (my url for ampache) returns '40 - Wrong username or password.'.
Relevant logs are 1).

2) Checking low security checkbox allows me to login and everything works except songs don't play. I can get metadata and everything else though.
Relevant logs are 2).

## Expected behavior
Successful login in case 1) without low security checked, and songs playback in case 2) with low security checked

## To Reproduce
1. Set up ampache via docker, go through initial configuration.
2. Edit Caddyfile from Caddy reverse proxy to point to ampache port.
3. Set up Tempo on android.
4. Connect without and with Low Security checked.

## Environment
 - Android device: Xiaomi Redmi Note 9 Pro
 - Android OS version: 12
 - App version: 3.8.1
 - Other relevant details: 

## Logs or Screenshots
1) These are the log sections from ampache when trying to connect normally

> 2024-10-11T08:54:59+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
> 2024-10-11T08:54:59+00:00 [ampache] (Ampache\Module\Api\SubsonicApiApplication) -> Invalid authentication attempt to Subsonic API for user [admin]
> 2024-10-11T08:54:59+00:00 [ampache] (Ampache\Module\Api\Subsonic_Xml_Data) -> API fail in function ping-1.16.1

2) Relevant log sections from ampache when trying to connect with low security checked

> 2024-10-11T08:30:01+00:00 [ampache] (Ampache\Module\System\Dba) -> Database connection...
>  2024-10-11T08:30:01+00:00 [admin] (Ampache\Module\System\Session) -> Session created: 44a2d0b3489c8a679f4f7ebe7bb73627
> 2024-10-11T08:30:01+00:00 [admin] (Ampache\Module\Api\Subsonic_Api) -> Stream proxy: https://ampache.tld:80/play/index.php?ssid=44a2d0b3489c8a679f4f7ebe7bb73627&type=song&oid=1&uid=1&client=Tempo&cache=18640063&transcode_to=mp3&bitrate=32000&player=api&name=Lezel%20-%20Mikansei%20Rendezvous.mp3
> 2024-10-11T08:30:01+00:00 [admin] (Ampache\Module\Api\Subsonic_Api) -> Stream error: OpenSSL/3.0.14: error:0A00010B:SSL routines::wrong version numbere\System\Session) -> 

## Additional context
This is my first time trying to setup ampache and trying tempo. I did try to setup substreamer client and it also doesn't connect, but if I enable Legacy signin option (iirc its name) it connects and plays songs fine.
I suspect this is somehow related to the version of subsonic used by ampache-docker (1.16.1) and the fact that with low security ampache returns port https://ampache.tld:80/play/whatever. If I enter the url with port 443 after domain the browser downloads the song.

**Comment by addy419** [comment]
@i-vis I have the same problem. While Ampache seems to be broken, Subsonic works fine. Not sure why?

Power Ampache 2 works fine with Ampache tho, so definitely a Tempo bug.

**Comment by i-vis** [comment]
Ok, so it was actually **user error.** I've been using ampache the way I described earlier for some months, it didn't bother me much, that's why it took me so long to figure out my mistake.
Essentially new clients **require API key of ampache to be used as password**.
So, if you have this problem:
1. Go to your Account settings on your ampache server
2. Copy API key (right under QR code). Regenerate if needed.
3. Use this API key instead of password to account.

This works with Tempo, Subtracks, Ultrasonic, Substreamer clients from my testing.
This problem is user error, although I wish I stumbled upon the solution earlier.
